src/modules/users/entities/role.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { Column, Entity, OneToMany, PrimaryColumn } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class Role {
+  @PrimaryColumn()
```

Comment:
To set strict data types in SQL database is a good practice.
not critical here, but keep in mind